src/main/java/service/Processor.java

```diff
@@ -1,47 +1,47 @@
 package service;
 
-import entity.Car;
+import entity.*;
 
 import java.util.Arrays;
 import java.util.NoSuchElementException;
 import java.util.Optional;
 
-public class Processor {
+public class Processor implements  IProcessor{
```

Comment:
If this class implement Processor interface it should have *Impl suffix - ProcessorImpl.